lua/cmake-tools/preset.lua

```diff
@@ -25,7 +25,10 @@ local function expandMacro(self, str)
   if self.generator then
     str = str:gsub("${generator}", self.generator)
   end
-  str = str:gsub("${hostSystemName}", vim.loop.os_uname().sysname)
+  str = str:gsub(
+    "${hostSystemName}",
+    vim.fn.has("win32") == 1 and "Windows" or vim.loop.os_uname().sysname
```

Comment:
I think a require for osys would be nice since it would be inline with how other code does this check. Either an "inline require" or one at the top of the file. Maybe something like this
`require("cmake-tools.osys").iswin32 and "Windows" or vim.loop.os_uname().sysname`
or
```
<at the top of the file>
local osys = require("cmake-tools.osys")
...
osys.iswin32 and "Windows" or vim.loop.os_uname().sysname
```